homeassistant/components/media_player/squeezebox.py

```diff
@@ -244,6 +275,11 @@ def state(self):
                 return STATE_IDLE
         return STATE_UNKNOWN
 
+    @property
+    def source_list(self):
+        """List of available sources (playlists)."""
+        return sorted(self._lms.playlists)
```

Comment:
Media player sources aren't the same as play lists. We shouldn't use this property for something it's not intended for.

Reply:
I see the same behaviour in other components (2 min search)

added in (#12045)
https://github.com/home-assistant/home-assistant/blob/f09cea149903365edff561c8d61d4bdd1fafcae8/homeassistant/components/media_player/volumio.py#L205-L209

https://github.com/home-assistant/home-assistant/blob/f09cea149903365edff561c8d61d4bdd1fafcae8/homeassistant/components/media_player/ziggo_mediabox_xl.py#L134-L139

Reply:
Existing bad behavior isn't a reason to introduce new bad behavior.